apps/api-journeys/src/lib/mockDb.ts

```diff
@@ -0,0 +1,16 @@
+import { PrismaClient } from '@prisma/client'
```

Comment:
```suggestion
import { PrismaClient } from '.prisma/api-journeys-client'
```

this will give you strong types.